cmake/herbert_FindMPI.cmake

```diff
@@ -2,9 +2,9 @@ set(EXTERNAL_ROOT "${Herbert_ROOT}/_LowLevelCode/external")
 if(UNIX)
     if (USE_HERBERT_MPI)
     	set(MPICH_VERSION "3.3a2")
-	set(MPI_ROOT "${EXTERNAL_ROOT}/glnxa64/mpich-${MPICH_VERSION}")
+	    set(MPI_ROOT "${EXTERNAL_ROOT}/glnxa64/mpich-${MPICH_VERSION}")
```

Comment:
unclear why you are increasing the indent at this point

Reply:
that's how web interface interprets tabs. in fact,  no additional indent . I've fixed the tabs